websocketOutportDriver/payloadParser.go

```diff
@@ -102,3 +105,48 @@ func padUint32ByteSlice(initial []byte) []byte {
 	padding := bytes.Repeat([]byte{0}, 4)
 	return append(padding, initial...)
 }
+
+func prepareArgsSaveBlock(args *outport.ArgsSaveBlockData) *outport.ArgsSaveBlockData {
+	prepareTxs := func(initial map[string]dataCore.TransactionHandlerWithGasUsedAndFee) map[string]dataCore.TransactionHandlerWithGasUsedAndFee {
+		res := make(map[string]dataCore.TransactionHandlerWithGasUsedAndFee, len(initial))
+		for txHash, tx := range initial {
+			res[hex.EncodeToString([]byte(txHash))] = tx
+		}
+		return res
+	}
+	prepareLogs := func(initial []*dataCore.LogData) []*dataCore.LogData {
```

Comment:
could have extracted 2 new function for prepareTxs and prepareLogs. The code would look better

Reply:
refactored